src/Core/Prefab/GlobeView.js

```diff
@@ -163,6 +164,31 @@ export function createGlobeLayer(id, options) {
         return false;
     }
 
+    // return common extent to pool the geometries
+    // the geometry in common extent is identical to the existing input
+    // with a transformation (translation, rotation)
+    wgs84TileLayer.getCommonGeometryExtent = (extent) => {
```

Comment:
These functions don't belong to `GlobeView` because:
- the panorama layers will want to use them
- they are tile geometries implementation details

Reply:
Also the naming is wrong: you're basically doing a modulo operation on the longitude component (why `sizeLatitude`?) of the extent so the name has to express this.
Btw, why aren't you doing the same thing for the other angle? (I guess it has to do with the ellipsoid shape, but you may want to add a comment in the code)

Reply:
* moved in tile builder
* sizeLatitude is modified
* getCommonGeometryExtent, in function of builder, it doesn't do modulo